src/TgSharp.Core/Network/Requests/AckRequest.cs

```diff
@@ -6,10 +6,12 @@
 
 namespace TgSharp.Core.Network.Requests
 {
-    public class AckRequest : TLMethod
+    public class AckRequest : TLObject
     {
         private readonly List<ulong> msgs;
 
+        public override int Constructor => 0x62d6b459;
```

Comment:
what is this magic number?

Reply:
Each TLObject-derived class has a constructor it's part of the Telegram Protocol. check other part of the code for more example.